prac_07/project_managment.py

```diff
@@ -0,0 +1,57 @@
+from prac_07.project import Project
+
+MENU_STRING = "(L)oad projects\n(S)ave project\n(D)isplay projects\n(F)ilter projects by date\n(A)dd new peojects\n(U)pdate projects\n(Q)uit"
+
+
+def main():
+    print(MENU_STRING)
+    choice = input(">>> ").upper()
+    while choice != "Q":
+        if choice == "L":
+            load_project()
+        elif choice == "S":
+            print("S")
+        elif choice == "D":
+            display_project(projects)
+        elif choice == "F":
+            print("filter")
+        elif choice == "A":
+            print("add")
+        elif choice == "U":
+            print("update")
+        else:
+            print("Invalid menu choice")
+            print(MENU_STRING)
+        choice = input(">>> ").upper
+
+
+def load_project():
+    in_file = open("projects.txt", "r")
```

Comment:
I find it helpful to use `with` so I don't forget to close files.

`with open("projects.txt", "r") as in_file:`